_docs/overview/key-concepts.md

```diff
@@ -13,14 +13,13 @@ padlocked box, of which there can be multiple padlocks - see the
 [Multi-signature Security](/docs/overview/multi-signature-security)
  section.
 
-Unlike a password, a private key is not meant for you to remember.
-It's a long string of gibberish.
-The private key is what you need to keep
-secure. If anyone gets it, they can take your money. Unlike traditional
-financial instruments, there is no recourse. There is no company that is
-liable, because Bitcoin is a decentralized system not run by any person or
-entity. And no law enforcement agency is likely to investigate your
-case.
+Unlike a password, a private key is not meant to be memorized. It is a
+256-bit number, usually expressed as a 64-character hexadecimal string.
```

Comment:
Glacier is targeted at non-technical people who are likely to understand "long string of gibberish" better than "64-character hexadecimal string". Perhaps this detail belongs in a footnote. (Once #13 enables footnotes again.)